PWGGA/GammaConv/AliAnalysisTaskGammaCalo.cxx

```diff
@@ -4061,7 +4069,10 @@ void AliAnalysisTaskGammaCalo::ProcessTrueClusterCandidates(AliAODConversionPhot
 
   if (TruePhotonCandidate->IsLargestComponentPhoton() || (TruePhotonCandidate->IsLargestComponentElectron() && TruePhotonCandidate->IsConversion())) {
     fHistoTrueClusGammaPt[fiCut]->Fill(TruePhotonCandidate->Pt(), tempPhotonWeight);
-    if (!fDoLightOutput && fDoClusterQA > 0) fHistoTrueClusGammaPtM02[fiCut]->Fill(TruePhotonCandidate->Pt(), clusterM02, tempPhotonWeight);
+    if (!fDoLightOutput && fDoClusterQA > 0){
+      fHistoTrueClusGammaPtM02[fiCut]->Fill(TruePhotonCandidate->Pt(), clusterM02, tempPhotonWeight);
+      fHistoTrueClusGammaEResPt[fiCut]->Fill((TruePhotonCandidate->E()-Photon->E())/Photon->E(), TruePhotonCandidate->Pt(), tempPhotonWeight);
```

Comment:
I think this will be distorted by the conversion electrons. Do you also want to implement the same histogram for pure photon clusters?

Reply:
Yes, I will do that!